From 7abe44a6c60f996fb9e66ad4b225477b715c2fb0 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 30 Jul 2023 10:28:14 +0200
Subject: [PATCH 402/464] arm64: dts: rk3566-pinetab2: Drop useless
 regulator-state-mem nodes

This is only useful for PMIC.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 .../boot/dts/rockchip/rk3566-pinetab2.dtsi    | 20 -------------------
 1 file changed, 20 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi
index 349071581962..f516670d0125 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-pinetab2.dtsi
@@ -173,10 +173,6 @@ vcc_3v3: vcc_3v3 {
 		regulator-min-microvolt = <3300000>;
 		regulator-max-microvolt = <3300000>;
 		vin-supply = <&vcc3v3_sys>;
-
-		regulator-state-mem {
-			regulator-off-in-suspend;
-		};
 	};
 
 	vcc3v3_minipcie: vcc3v3_minipcie {
@@ -189,10 +185,6 @@ vcc3v3_minipcie: vcc3v3_minipcie {
 		regulator-min-microvolt = <3300000>;
 		regulator-max-microvolt = <3300000>;
 		vin-supply = <&vcc_sys>;
-
-		regulator-state-mem {
-			regulator-off-in-suspend;
-		};
 	};
 
 	vcc3v3_sd: vcc3v3_sd {
@@ -204,10 +196,6 @@ vcc3v3_sd: vcc3v3_sd {
 		regulator-min-microvolt = <3300000>;
 		regulator-max-microvolt = <3300000>;
 		vin-supply = <&vcc3v3_sys>;
-
-		regulator-state-mem {
-			regulator-off-in-suspend;
-		};
 	};
 
 	vcc5v0_usb_host0: vcc5v0_usb_host0 {
@@ -220,10 +208,6 @@ vcc5v0_usb_host0: vcc5v0_usb_host0 {
 		regulator-min-microvolt = <5000000>;
 		regulator-max-microvolt = <5000000>;
 		vin-supply = <&vcc5v_midu>;
-
-		regulator-state-mem {
-			regulator-off-in-suspend;
-		};
 	};
 
 	vcc5v0_usb_host2: vcc5v0_usb_host2 {
@@ -236,10 +220,6 @@ vcc5v0_usb_host2: vcc5v0_usb_host2 {
 		regulator-min-microvolt = <5000000>;
 		regulator-max-microvolt = <5000000>;
 		vin-supply = <&vcc5v_midu>;
-
-		regulator-state-mem {
-			regulator-off-in-suspend;
-		};
 	};
 
 	vcc_bat: vcc_bat {
-- 
2.34.1

